samples/java-eventsourced-customer-registry/src/main/java/customer/MainWithCustomViewId.java

```diff
@@ -0,0 +1,34 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+
+package customer;
+
+import com.akkaserverless.javasdk.AkkaServerless;
+import customer.domain.CustomerEntity;
+import customer.domain.CustomerEntityProvider;
+import customer.view.CustomerByNameView;
+import customer.view.CustomerByNameViewProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class MainWithCustomViewId {
+
+  private static final Logger LOG = LoggerFactory.getLogger(MainWithCustomViewId.class);
+
+  // tag::register[]
+  public static AkkaServerless createAkkaServerless() {
+    AkkaServerless akkaServerless = new AkkaServerless();
+    akkaServerless.register(CustomerByNameViewProvider.of(CustomerByNameView::new)
+        .withViewId("CustomerByNameV2"));
```

Comment:
since the docs are telling users to keep the old view, we could also show it here, for instance: 

```suggestion
    // TODO: remove this registration on next deployment 
    // and after verifying that new view is working
    akkaServerless.register(CustomerByNameViewProvider.of(CustomerByNameView::new)

    akkaServerless.register(CustomerByNameViewProvider.of(CustomerByNameView::new)
        .withViewId("CustomerByNameV2"));
```

Reply:
Then I would have to add another `CustomerByNameViewV2` to the sample and we should have registration code for both steps of the rollout. As I wrote the steps this code is only referred to in the final step so it's ok.

Could be good to elaborate on that process in more detail, but I'll leave that for later.

Reply:
ok, that's fine.

Reply:
created issue https://github.com/lightbend/akkaserverless-java-sdk/issues/348 for that